scripts_and_studies/coregistration_study/pyraws_coregistration_profiling.ipynb

```diff
@@ -69,7 +69,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "test_files=sorted(glob(os.path.join(\"tests\",test_type, \"*\")))"
+    "test_files=sorted(glob(f\"coregistration_profiling_results\\\\{test_type}\\\\*.txt\"))"
```

Comment:
```suggestion
    "test_files=sorted(glob(os.path.join("coregistration_profiling_results", test_type, "*")))"
```

Reply:
The error is not in the use of os.path.join() but in the directory name (i.e., "tests" instead of "coregistration_profiling_results"). The syntax that you wrote breaks the compatibility with Linux.